### DISCUSSION TITLE
Unable to get into preview program?

### DISCUSSION THREAD

**Discussion by viridivn** [microsoft/WSA#186] [General]
I'm in the Windows dev channel, and I'm trying to join the preview program for WSA to get the new android 13 build - I tried submitting the form on MS' website (submitting MSA email address), but never got any confirmation email, nor does the Appstore or Windows Store show any updates. What else can I try?

**Comment by infinitepower18** [comment]
It takes a few days for it to be processed

**Comment by sn-o-w** [comment]
I’m facing the same issue. I submitted the form about two weeks ago and never got any confirmation email. 🤔

**Comment by CAESFIR** [comment]
Same issue here, and it's Septempter 12th 2023 now.

**Reply by sn-o-w** [reply]
In my case, I was added to the WSA Preview program eventually, but I got zero emails. 😂

**Comment by Darkhost1999** [comment]
I have tried at least 5 times over the past several months. No luck 😪

I'm dev insider
Microsoft store insider
Xbox insider
Have WSL pre-release
Just no WSA pre-release

I haven't exactly been able to make use of WSA enjoyably at all this entire time. And I'm not sure a preview will fix anything.
